src/Components/HomePage/Sections/KnowAbout.js

```diff
@@ -31,7 +31,7 @@ class KnowAbout extends Component {
             <span className={styles.About}> About </span>
           </h2>
           <Row
-            className="justify-content-md-center"
+            className={styles.justify_content}
```

Comment:
This change has centered cards in mobile view but left aligned them in desktop view. Please check.